cmake/WebPConfig.cmake.in

```diff
@@ -1,6 +1,4 @@
 @PACKAGE_INIT@
 
-set(WebP_INCLUDE_DIRS "webp")
-set(WEBP_INCLUDE_DIRS ${WebP_INCLUDE_DIRS})
-set(WebP_LIBRARIES "@INSTALLED_LIBRARIES@")
-set(WEBP_LIBRARIES "${WebP_LIBRARIES}")
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
```

Comment:
I think we need `find_package` for dependencies here, example:
* https://github.com/elucideye/drishti/blob/2ca66a7e0c92d1c1c382774b5f1ff9c34e192635/cmake/Config.cmake.in#L3-L7

Functionality similar to `find_dependency` macro but `find_dependency` will not work with `CONFIG` in CMake 3.0:
* https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

Reply:
@ruslo : fixed.